docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -402,24 +402,21 @@ Prefer negated matchers over `to change.by(0)`.
 [source,ruby]
 ----
 # bad
-expect { run }.to change(Foo, :bar).by(0)
-expect { run }.to change { Foo.bar }.by(0)
-expect { run }
-  .to change(Foo, :bar).by(0)
-  .and change(Foo, :baz).by(0)
```

Comment:
I believe that the multiline form makes it easier to spot that `bar` and `baz` are different attributes.

Reply:
Indeed it is. It looks better than making it a single line.